pkg/tasks/queue/pgqueue.go

```diff
@@ -721,11 +721,14 @@ func (p *PgQueue) ListenForCancel(ctx context.Context, taskID uuid.UUID, cancelF
 
 	// Register a channel for the task where a notification can be sent to cancel the task
 	channelName := getCancelChannelName(taskID)
+	// TODO remove debug logs. checking if channel register is hanging
+	logger.Debug().Msg("ListenForCancel: preparing register channel")
 	_, err = conn.Conn().Exec(ctx, "listen "+channelName)
 	if err != nil {
 		logger.Error().Err(err).Msg("ListenForCancel: error registering channel")
 		return
 	}
+	logger.Debug().Msg("ListenForCancel: finished register channel")
```

Comment:
would debug even show up in kilbana?  wouldn't it need to be warn?

Reply:
in stage i mean

Reply:
I see all the log levels when I  search through kibana

Reply:
oh interesting!  ACK